<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataManage</class>
 <widget class="QMainWindow" name="DataManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataManage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>1031</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>主组</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>971</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>561</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>查找</string>
     </property>
     <widget class="QPushButton" name="findBtn">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>90</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="findKey1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>最小时间</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="findKey4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>名称</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="findKey3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="findName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>关键词</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="findKey2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="findMinYear">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="findMaxYear">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>最大时间</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showAllBtn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>显示全表</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="submitAlterBtn">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>510</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeAlterBtn">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>510</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>撤销修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="insertBtn">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>510</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>添加记录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBtn">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>510</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>删除选中行</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
